lib/dark_worlds_server/runner_supervisor/runner.ex

```diff
@@ -228,47 +223,14 @@ defmodule DarkWorldsServer.RunnerSupervisor.Runner do
     {:stop, {:shutdown, :game_timeout}, state}
   end
 
-  def handle_info({:spawn_bots, bot_count}, state) when bot_count > 0 do
-    {:ok, bot_handler_pid} = BotPlayer.start_link(self(), %{})
-
-    {game_state, bots_ids} =
-      Enum.reduce(0..(bot_count - 1), {state.game_state, []}, fn _, {acc_game_state, bots} ->
-        # character = Enum.random(["h4ck", "muflus"])
-        character = "muflus"
-        {new_game_state, player_id} = GameBackend.add_player(acc_game_state, character)
-
-        {new_game_state, [player_id | bots]}
-      end)
-
-    Process.send_after(self(), {:activate_bots, bots_ids}, 10_000)
-
-    state =
-      Map.put(state, :game_state, game_state)
-      |> Map.put(:bot_handler_pid, bot_handler_pid)
-
-    NewRelic.increment_custom_metric("GameBackend/TotalBots", bot_count)
```

Comment:
Don't we need this metric? maybe we could add it to the bot player init function

Reply:
Need might be a strong word :rolf:, but I've added it back in `bot_client`

Reply:
hahaha i'll rephrase to "should we keep this metric?"